near/src/config.rs

```diff
@@ -172,7 +172,7 @@ impl NearConfig {
         config: Config,
         genesis_config: &GenesisConfig,
         network_key_pair: KeyFile,
-        block_producer: Option<&BlockProducer>,
+        block_producer: Option<BlockProducer>,
```

Comment:
Why not use a reference?

Reply:
Because there is no point - it gets copied inside anyway. And there is no value in having ownership of copy outside of this - so passing the value here.